tests/scripts/perf-prep.sh

```diff
@@ -64,19 +64,17 @@ python3.5 --version
 python3.5 ./tests/scripts/Microsoft.BenchView.JSONFormat/tools/machinedata.py
 
 if [ $throughput -eq 1 ]; then
-    # Clone corefx
-    if [ -d "_" ]; then
-        rm -r -f _
+    # Download throughput benchmarks
+    if [ -d "Microsoft.Benchview.ThroughputBenchmarks.x64.Windows_NT" ]; then
+        rm -r -f Microsoft.Benchview.ThroughputBenchmarks.x64.Windows_NT
```

Comment:
Gotcha. Yes, I think it would be better to either parametrize it out, *or* get rid of it altogether (if possible we should ensure we use the same input set for all platforms, so that we can compare x64-vs-x86 just like windows-vs-linux). But, no need to hold up this PR for that change, as long as the code behaves as intended (it makes mostly sense, since we only have x64 on Linux).